pages/glossary/accessible-name.md

```diff
@@ -15,7 +15,7 @@ For native markup languages, such as HTML and SVG, additional information on how
 
 **Note:** As per the [accessible name and description computation](https://www.w3.org/TR/accname), accessible names are [flat string](https://www.w3.org/TR/accname-1.1/#terminology) trimmed of leading and trailing whitespace. Notably, it is not possible for a non-empty accessible name to be composed only of whitespace since these must be trimmed.
 
-#### Accessibility Support
+#### Accessibility Support for Definition of Accessible name
```

Comment:
It would be better if you use small letter for word "Definition" and/or "Accessible".
Suggestion is to use one of the following: 
- Accessibility Support for defination of _accessible name_
- Accessibility Support for defination of Accessibile Name